src/libclipper/src/redis.cpp

```diff
@@ -438,6 +454,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
```

Comment:
Great catch :) I think it's correct to do this in `redis.cpp`, but we could also do it in the management front end. Just want to confirm that `redis.cpp` is the right place for it - thoughts?